### PR TITLE
Sync "linux-source" in non-Windows host

### PR DESCRIPTION
**Pull request by BackSlasher**

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -9,10 +9,9 @@ Vagrant.configure("2") do |config|
 
 	# The following will not work on case-insensitive filesystems like those provided by
 	# Windows and macOS, as the Linux kernel source tree has files that differ only by case.
-	# If your host is Linux, you can uncomment this line to share the kernel source
-	# directory with your host for easy editing.
-
-	# config.vm.synced_folder "./linux-source", "/home/vagrant/linux-source"
+        unless Vagrant::Util::Platform.windows?
```
Comment (idoby) [inline Vagrantfile]: Better to use fs_case_sensitive, I think

http://www.rubydoc.info/github/mitchellh/vagrant/Vagrant/Util/Platform#fs_case_sensitive%3F-class_method

Comment (BackSlasher) [reply Vagrantfile]: I don't like the hacky way that method works, but surewhynot

Comment (BackSlasher) [reply Vagrantfile]: fixed

File: Vagrantfile [hunk 2]
```diff
@@ -9,10 +9,9 @@ Vagrant.configure("2") do |config|
 
 	# The following will not work on case-insensitive filesystems like those provided by
 	# Windows and macOS, as the Linux kernel source tree has files that differ only by case.
-	# If your host is Linux, you can uncomment this line to share the kernel source
-	# directory with your host for easy editing.
-
-	# config.vm.synced_folder "./linux-source", "/home/vagrant/linux-source"
+        unless Vagrant::Util::Platform.fs_case_sensitive?
```
Comment (idoby) [inline Vagrantfile]: Should be "if", right?

File: Vagrantfile [hunk 3]
```diff
@@ -9,7 +9,7 @@ Vagrant.configure("2") do |config|
 
 	# The following will not work on case-insensitive filesystems like those provided by
 	# Windows and macOS, as the Linux kernel source tree has files that differ only by case.
-        unless Vagrant::Util::Platform.fs_case_sensitive?
+        if Vagrant::Util::Platform.fs_case_sensitive?
```
Comment (BackSlasher) [inline Vagrantfile]: Now it's backwards  :confused: 
`unless` == `if not`
Right now, your logic is "if platform is case sensitive, enable the synced folder".
You might want to change it back to `unless`, or to `if not` if it makes your more comfortable

Comment (idoby) [reply Vagrantfile]: It's ok, it requires the fs to be case-sensitive to work

**Pull request merged**